DepositeCalcTests/Tests/SettingsPageTests.cs

```diff
@@ -0,0 +1,165 @@
+            var settingsPage = new SettingsPage(driver);
+            string expectedUrl = "https://localhost:5001/";
+
+            // Act
+            settingsPage.ClickOnLogoutLnk();
+
+            // Assert
+            Assert.AreEqual(expectedUrl, driver.Url, "Incorrect page");
+        }
+
+        [Test]
+        public void CancelBtnTest()
+        {
+            // Arrange
+            var settingsPage = new SettingsPage(driver);
+            string expectedUrl = "https://localhost:5001/Calculator";
+
+            // Act
+            settingsPage.ClickOnCancelBtn();
+
+            // Assert
+            Assert.AreEqual(expectedUrl, driver.Url, "Incorrect page");
+        }
+
+        [TestCase("dd/MM/yyyy", "2024", "February", "29", "02")]
+        [TestCase("dd-MM-yyyy", "2026", "March", "01", "03")]
+        public void DateFormatStrartWithDayTest(string expectedDateFormat, string year, string month, string day, string expectedMonthNumber)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var settingsPage = new SettingsPage(driver);
+
+            // Act
+            settingsPage.DateFormatDropDown = expectedDateFormat;
+            settingsPage.ClickOnSaveBnt();
+            driver.SwitchTo().Alert().Accept();
+            calculatorPage.StartDateYear = year;
+            calculatorPage.StartDateMonth = month;
+            calculatorPage.StartDateDay = day;
+            string expectedSeparator = expectedDateFormat.Substring(2, 1);
+            string expectedEndDayStartsWithDays = day + expectedSeparator + expectedMonthNumber + expectedSeparator + year;
+
+            // Assert
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(expectedSeparator, calculatorPage.EndDate.Substring(2, 1), "Incorrect separator in the end date field");
+                Assert.AreEqual(expectedEndDayStartsWithDays, calculatorPage.EndDate, "Incorrect date format in the end date field");
+            });
+        }
+
+        [TestCase("MM/dd/yyyy", "2024", "April", "10", "04")]
+        [TestCase("MM dd yyyy", "2024", "September", "25", "09")]
+        public void DateFormatStrartWithMonthTest(string expectedDateFormat, string year, string month, string day, string expectedMonthNumber)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var settingsPage = new SettingsPage(driver);
+
+            // Act
+            settingsPage.DateFormatDropDown = expectedDateFormat;
+            settingsPage.ClickOnSaveBnt();
+            driver.SwitchTo().Alert().Accept();
+            calculatorPage.StartDateYear = year;
+            calculatorPage.StartDateMonth = month;
+            calculatorPage.StartDateDay = day;
+            string expectedSeparator = expectedDateFormat.Substring(2, 1);
+            string expectedEndDayStartsWitMonth = expectedMonthNumber + expectedSeparator + day + expectedSeparator + year;
```

Comment:
Suggestion: use the same start date and other numbers in test; and add 'expectedEndDate' to the test parameters.